src/java.base/share/classes/java/io/ObjectStreamField.java

```diff
@@ -164,12 +164,6 @@ public String getName() {
     @SuppressWarnings("removal")
     @CallerSensitive
     public Class<?> getType() {
```

Comment:
This method is no longer caller sensitive, also no need to import ReflectUtil now.